pyflakes/checker.py

```diff
@@ -1155,6 +1162,10 @@ def handleNodeLoad(self, node):
                     continue
 
             binding = scope.get(name, None)
+
+            if getattr(binding, 'assigned', None) is False:
```

Comment:
ah -- my point on this was more that this shouldn't be something we're guessing at -- it should always have this attribute or never have this attribute (it should be set unconditionally on construction)